data/dataPaths.json

```diff
@@ -617,7 +617,7 @@
       "enchantments": "pc/1.13.2",
       "version": "pc/1.15",
       "blocks": "pc/1.15.2",
-      "biomes": "pc/1.15.2",
+      "biomes": "pc/1.15",
```

Comment:
This seems to be incorrect isn't it ?
It seems you updated that file too

Reply:
I've put a biome file in every major version, so 1.15 have its own biomes.json. How is it incorrect?

Reply:
Ah right, I misread this, I thought this was for 1.15.2